### DISCUSSION TITLE
Welcome to tima-r Discussions!

### DISCUSSION THREAD

**Discussion by Adafede** [taxonomicallyinformedannotation/tima#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qfavregod** [comment]
Hi Adriano! Thank you for the amazing job you have done so far. I am facing an error when tryin to run Rscript to install.R: 

Current version is the latest, no need to update.
✔ Loading metadata database ... done
✖ Creating lockfile pkg.lock [8.1s]
Error:
! error in pak subprocess
Caused by error in `readRDS(dbfile)`:
! error reading from connection
---
Backtrace:
1. pak::lockfile_create()
2. pak:::remote(function(...) { …
3. err$throw(res$error)
---
Subprocess backtrace:
 1. base::withCallingHandlers(cli_message = function(msg) { …
 2. get("lockfile_create_internal", asNamespace("pak"))(...)
 3. prop$solve()
 4. private$plan$solve(policy = private$policy)
 5. pkgdepends:::pkgplan_solve(self, private, match.arg(policy))
 6. self$resolve()
 7. pkgdepends:::pkgplan_resolve(self, private)
 8. pkgdepends:::synchronise(self$async_resolve())
 9. asNamespace("pkgcache")$synchronise(...)
10. base::stop(priv$value)
11. global (function (e) …
Execution halted

Any suggestions... Thx

**Reply by Adafede** [reply]
Hey Quentin! Thank you for the compliments. Looks like a weird error there is not much I could do to reproduce. Could you try running the command again? Maybe some process got stuck by pure unluck...disk space, internet connexion or else?

**Comment by adrien-jagora** [comment]
Hi Adriano! I've followed your advice and created a Github account for the occasion ! I had one additionnal question regarding the optional MS1-based annotation on Lotus. If I understand correctly, this happens when the score S1 equals 0 as a way to allow TimaR to do its taxonomical annotation later on. Is this MS1 dereplication based on the exact mass of the precursor ion considered or on a molecular formula previously generated ? Thanks in advance !

**Reply by Adafede** [reply]
Hi @adrien-jagora,

Nice to see you created a GitHub account!

Regarding your question, best is if you ask it on a separate thread so that people can search for it later and not in here as it might not be the best for findability.

I'll open one for you.

**Reply by adrien-jagora** [reply]
Ok, thanks !

**Comment by Y-Thom** [comment]
Dear Dr. Rutz
 I started a job on TimaR with annotations obtained on sirius 6 and I get an error message.
No problem running the same job without the .zip folder which contains all the .tsv or .csv files obtained after the sirius 6 “all hits” summaries, but when I load the .zip and run it, the script stops on this error: 
Warning: Error in : Error in tar_make():
  `file` must be a length one character vector or numeric
  See https://books.ropensci.org/targets/debugging.html

Attached is the R output after launching the app.
[runApp().txt](https://github.com/user-attachments/files/20598861/runApp.txt)

Do I have a format problem with the Sirius annotation files? 

Many thanks for your help,

Best regards,

[runApp().txt](https://github.com/user-attachments/files/20598864/runApp.txt)

**Reply by Adafede** [reply]
Dear @Y-Thom,

Thank you for the detailed logs.

Could you also send the path you provided for your Sirius job? And the content of the folder? Hopefully this will help finding the issue. 

Best,

Adriano

**Reply by Y-Thom** [reply]
Thank you for your fast reply !
My Sirius data summaries are located here : "C:\Users\CEVAMAP\GitHub\tima-main\data\interim\PYRAMIMONAS.zip"
And here is the content of the .zip
https://www.grosfichiers.com/Mr6N3eg5YVy

Thank you for your time and help,

Best regards,

**Reply by Adafede** [reply]
Hi again,

It looks like [canopus](https://bio.informatik.uni-jena.de/software/canopus/) summaries are missing from your SIRIUS results. This is highly likely the reason of your job failing with SIRIUS results.

Since TIMA is using the chemical classes for reranking, they are not optional (yet, at least). You can obtain them with the `classes` flag as indicated in
https://taxonomicallyinformedannotation.github.io/tima/articles/II-preparing.html#sirius

Hope this helps,

Adriano

**Reply by Y-Thom** [reply]
Hello, 
Thank you for your help, it was impossible to get the canopus file with the version of sirius I was using (6.0.1) but with the new version ( 6.2.2) I no longer have this problem. Thanks a lot!
